pkg/kube/deployments.go

```diff
@@ -428,6 +429,14 @@ func GetContainerPort(deployment *appsv1.Deployment) map[int]int {
 	}
 }
 
+func GetContainerPortForDeploymentConfig(deploymentConfig *appv1.DeploymentConfig) map[int]int {
```

Comment:
Maybe move this function to `deploymentconfigs.go` ?

Reply:
Updated in 35a70e4f09ff5dd2024d03dca95b910c5c764493